ui/packages/consul-ui/app/templates/dc/services/instance.hbs

```diff
@@ -101,8 +101,13 @@ as |item|}}
           {{! so if we can ever get ServiceID from elsewhere we could save }}
           {{! a HTTP request/long poll here }}
           {{#if meta.data.ServiceID}}
+          {{! if we have a proxy then get the additional instance information }}
+          {{! for the proxy itself so if the service is called `backend` }}
+          {{! its likely to have a proxy service called `backend-sidecar-proxy` }}
+          {{! and this second request get the info for that instance and saves }}
+          {{! it into the `proxy` variable }}
             <DataSource
-              @src={{uri '/${partition}/${nspace}/${dc}/proxy-service-instance/${id}/${node}/${name}'
+              @src={{uri '/${partition}/${nspace}/${dc}/service-instance/${id}/${node}/${name}'
```

Comment:
We removed `proxy-` here as now we don't need to do the ServiceInstance/ProxyInstance munging we can just as for the service information of the proxy using the same endpoint as retrieving the service information for the service itself.